hotel-api-services/src/main/java/com/fponce/hotelapp/hotel/HotelController.java

```diff
@@ -0,0 +1,34 @@
+package com.fponce.hotelapp.hotel;
+
+import com.fponce.hotelapp.hotel.service.HotelService;
+import com.hotelapp.hotelapp.model.Hotel;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.inject.Inject;
+import javax.validation.Valid;
+import java.sql.SQLException;
+
+@RestController
```

Comment:
Is it correct to use `@RestController` here?
As we're building an API, I think we could consider using `@GetMapping | @PostMapping | @DeleteMapping`

Reply:
Hey @ragore09 do you mean it is not necessary to add ```@RestController``` here? Or should I only use ```@PostMapping```?

Reply:
my bad, I meant `RequestMapping`
what's the default HTTP Method for `RequestMapping`?
would it be useful to use instead  `@GetMapping | @PostMapping | @DeleteMapping | @PatchMapping` of just `@RequestMapping`?

Reply:
```RequestMapping``` has no default HTTP method mapping, so if we don’t specify a value, it’s going to map to any HTTP request. I annotated the Controller Class with ```@RestController``` and the method with ```@PostMapping``` so it maps to Post method.